config/example/custom_items.txt

```diff
@@ -1,41 +1 @@
-##
```

Comment:
what, why are you removing the comments from the start of the txt, don't do that

Reply:
It's from the EXAMPLE text. This wasn't even supposed to be committed with the revolver fix, so I just deleted all of it so it don't affect the main repository after a merge or anything.

Reply:
what

putting up a PR to remove all the lines from a file is very much going to effect the main repository if it's merged

Reply:
You pushed the deletion of the example text on your local desktop code to the branch you're working on. Merging this will delete the example text files in the actual code.

Reply:
Oh for pete's sake. Well then how do we fix this.